packages/devkit/src/utils/add-plugin.ts

```diff
@@ -232,17 +232,20 @@ function processProject(
   if (!tree.exists(packageJsonPath)) {
     return;
   }
-  const packageJson = readJson<PackageJson>(tree, packageJsonPath);
+  let packageJson = readJson<PackageJson>(tree, packageJsonPath);
```

Comment:
Nit: Is this change left behind from some older iteration of this code? It doesn't seem to serve a purpose?

Reply:
It is